vsts/node-release.yaml

```diff
@@ -0,0 +1,55 @@
+name: Release_$(BuildID)_$(Date:yyyyMMdd)$(Rev:.r)
+resources:
+- repo: self
+  clean: true
+
+phases:
+
+- phase: Phase_1
+  displayName: Create Release Branch
+
+  condition: succeededOrFailed()
+  queue:
+    name: Hosted Ubuntu 1604
+
+  steps:
+
+  - checkout: self  # self represents the repo where the initial Pipelines YAML file was found
+    persistCredentials: true  # set to 'true' to leave the OAuth token in the Git config after the initial fetch
+    clean: true
+
+  - task: NodeTool@0
+    inputs:
+      versionSpec: '10.x'
+
+  - script: |
+       npm install --global node-gyp
+       npm install --global lerna
+    displayName: 'Install Dependencies'
+    
+  - script: |
+        lerna bootstrap
+        lerna run build
+    displayName: 'Bootstrap & Build'
```

Comment:
i believe you need to run this after bump version otherwise the package-lock files aren't going to be good (that's why my comment was strategically placed after bump version but before npm pack). In other words:

1. install dependencies
2. bump version
3. lerna bootstrap
4. lerna run build
5. npm pack
6. publish artifacts